swift/ql/lib/codeql/swift/frameworks/StandardLibrary/Set.qll

```diff
@@ -0,0 +1,15 @@
+private import codeql.swift.dataflow.ExternalFlow
+
+/**
+ * A model for `Set` and related class members that permit data flow.
+ */
+private class SetSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        ";Set;true;insert(_:);;;Argument[-1].SetElement;ReturnValue.TupleElement[1];value",
+        ";Set;true;insert(_:);;;Argument[0];Argument[-1].SetElement;value",
```

Comment:
Should there be value flow from `Argument[0]` to `ReturnValue.TupleElement[1]` as well?

Reply:
Yes, there should